merkle/log_proofs.go

```diff
@@ -398,8 +400,7 @@ func skipMissingLevels(snapshot, lastNode int64, level int, node int64) (int, in
 			glog.Infof("Move down: S:%d L:%d LN:%d", sibling, level, lastNode)
```

Comment:
While you're changing this can you make this a verbose only log message, seems it was overlooked before.

Reply:
It checks `vvLevel` in the line above.